services/etl/requirements.txt

```diff
@@ -1,118 +1,11 @@
+aiohttp==3.8.3
+apache-airflow==2.5.1
```

Comment:
is the airflow import required? it should already be there with the docker image right?

Reply:
right, will remove